backend/dal/requestDAL.go

```diff
@@ -33,3 +45,28 @@ func (dal *RequestDAL) GetLastRequests(before time.Time, size int) ([]models.Req
 
 	return reqs, nil
 }
+
+// InsertRequest Receives a request as a parameter and inserts into the database
+func (dal *RequestDAL) InsertRequest(request models.Request) error {
+	insertQuery := `INSERT INTO request (name, description, extrainfo, minprice, maxprice, userid, categoryid, timestamp) 
+						VALUES ($1, $2, $3, $4, $5, $6, $7, $8)`
+
+	// Gets the controller of the database and executes the query
+	fmt.Println(request.Name)
+	fmt.Println(request.Description)
+	fmt.Println(request.ExtraInfo)
+	fmt.Println(request.MinPrice)
+	fmt.Println(request.MaxPrice)
+	fmt.Println(request.Userid)
+	fmt.Println(request.Categoryid)
+	fmt.Println(request.Timestamp)
+
+	_, err := dal.db.Exec(insertQuery, request.Name, request.Description, request.ExtraInfo, request.MinPrice, request.MaxPrice, request.Userid, request.Categoryid, request.Timestamp)
+
+	// Checks if any error happened during the query execution
+	if err != nil {
+		panic(err)
```

Comment:
If there's an error in the db, the whole server will stop. Maybe it'd better to rise the error to the controller or the handler and let the handler treat it. We could send a message to the frontend with the error, for example.

Reply:
Changed panic into return err, this way the handler will treat the error.